small patch to reflect simpler enablement for 7.69 (#31341)
DataDog/documentation: content/en/agent/fleet_automation/remote_management.md

Users must have the [Agent Upgrade][2] within Fleet Automation for upgrades, and the [Fleet Policies Write][2] permissions to configure Agents remotely. The permission is enabled by default on the Datadog Admin role.

## Enable Remote Agent Management

> If you are running Agent version 7.69+:
> 1. Ensure that your Agent configuration includes the environment variable `DD_REMOTE_UPDATES=true` set. This alone enabled Remote Agent Management on the Agent.

Reviewer: ```suggestion
If you are running Agent version 7.69+:
1. Ensure that your Agent configuration includes the environment variable `DD_REMOTE_UPDATES=true`. This enables Remote Agent Management on the Agent.
```

1. Use the generated Agent installation command to upgrade your Agent to version 7.66+.

> <div class="alert alert-info">For Agent versions older than 7.69, uou must run the generated installation command with <code>DD_REMOTE_UPDATES</code> set to <code>true</code> to gain access to Remote Agent Management. Enabling Remote Agent Management without running the installation command does not grant access to the feature.</div>

Reviewer: ```suggestion
<div class="alert alert-info">For Agent versions prior to 7.69, you must run the generated installation command with <code>DD_REMOTE_UPDATES</code> set to <code>true</code> to gain access to Remote Agent Management. Enabling Remote Agent Management without running the installation command does not grant access to the feature.</div>
```
